architecture/overview.adoc

```diff
@@ -5,22 +5,68 @@
 :icons:
 :experimental:
 
-OpenShift is a layered system designed to expose underlying Docker and Kubernetes concepts as accurately as possible, with a focus on easy composition of applications by a developer (install Ruby, push code, add MySQL). Unlike 2.x, more flexibility of configuration is exposed after creation in all aspects of the model. The concept of an application as a separate object is being removed in favor of more flexible composition of "services" - allowing two web containers to reuse a DB, or expose a DB directly to the edge of the network.
+OpenShift is a layered system designed to expose underlying
+https://www.docker.com/[Docker] and
+http://kubernetes.io/[Kubernetes] concepts as accurately as possible, with a focus on easy composition
+of applications by a developer (install Ruby, push code, add MySQL). Unlike
+OpenShift 2.x, more flexibility of configuration is exposed after creation
```

Comment:
We should probably say "v2" instead of "2.x", referring to the architectural version of OpenShift (since this applies to both Enterprise and Origin). See the Note in http://docs.openshift.org/latest/whats_new/overview.html

Alternatively, we could conditionalize these paragraphs and get specific with the reference (i.e., OSE 2.x / Origin M4).